.clang-format

```diff
@@ -41,7 +41,7 @@ BreakBeforeBraces: Mozilla
 BreakBeforeInheritanceComma: true
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: false
-BreakConstructorInitializers: BeforeComma
+BreakConstructorInitializers: AfterColon
```

Comment:
The appearance of simple constructors did not look good with former setting:
Before:
```
    Point()
      : x(0)
      , y(0)
      , z(0)
    {
    }

```
After:
`Point() : x(0), y(0), z(0) {}`